packages/core/src/Layout/Layout.tsx

```diff
@@ -173,15 +161,18 @@ const Layout: React.FC<InferProps<typeof propTypes>> = ({
       {React.Children.map(
         children,
         (child, idx) =>
-          child && (
+          child &&
+          React.isValidElement(child) && (
             <ZIndexBox
               width={widths[idx]}
               px={boxPaddingX}
               py={boxPaddingY}
               data-testid={`box-${idx}`}
-              zIndex={child.props.zIndex}
+              // TS ignored because the type of child.props is unknown
+              // @ts-ignore
+              zIndex={child?.props?.zIndex}
```

Comment:
(n-b) Does this need to be `undefined` to avoid collisions with `0`/falsy values?
```suggestion
              zIndex={child?.props?.zIndex || undefined}
```

Reply:
I had to check, but this should evaluate to `undefined` as it is. I'm not sure I actually _needed_ to add the optional chaining now that I'm looking at it again.
<img width="177" alt="Screenshot 2023-12-26 at 5 04 40 PM" src="https://github.com/priceline/design-system/assets/3260642/4eece6b2-793e-40e6-b6ae-c063c7f15d72">